requirements_dev.txt

```diff
@@ -0,0 +1,3 @@
+pytest==3.2.2
+pyIATI==0.2.0
```

Comment:
This is required for the app itself to run in prod, so should be in `requirements.txt`.